src/devices/vdml_imu.c

```diff
@@ -97,15 +105,42 @@ euler_s_t imu_get_euler(uint8_t port) {
 }
 
 double imu_get_pitch(uint8_t port) {
-	return imu_get_euler(port).pitch;
+	double rtn = PROS_ERR_F;
+	if (!claim_port_try(port - 1, E_DEVICE_IMU)) {
+		return rtn;
+	}
+	euler_s_t euler_values;
+	v5_smart_device_s_t* device;
+	device = registry_get_device(port - 1);
```

Comment:
Why split the definition and where you set it?

Reply:
Honestly, I did it because I saw that somebody else did it in another portion of the code so I thought it was the precedent for the project :/. Was wondering the same thing as you.